packages/react-compose/package.json

```diff
@@ -31,7 +31,7 @@
     "react": "^16.3.2"
   },
   "devDependencies": {
-    "@types/enzyme": "^3.1.10"
+    "@shopify/react-testing": "^1.7.3"
```

Comment:
I'd prefer we not declare this dependency if possible, I think it means react-testing bumps unnecessarily bump all these other libraries too